src/pages/item_templates/ItemTemplatePrimaryMedia.jsx

```diff
@@ -101,12 +107,30 @@ const ItemTemplatePrimaryMedia = observer(() => {
             ]}
           />
       }
-      <Inputs.Checkbox
+
+      <br/>
+      <Title order={3} mb="md">{ l10n.categories.bundled_media_property }</Title>
+
+      <Inputs.Select
         {...inputProps}
-        {...l10n.media.generative}
-        field="generative"
-        defaultValue={false}
+        {...l10n.additional_media.bundled_property}
+        field="bundled_property_id"
+        options={mediaProperties}
       />
+
+      <Inputs.Select
```

Comment:
You can actually change this to an <Inputs.Hidden> type, that element is precisely for this case where we just want to set a default value on render.